### PR TITLE
[SEO alt text]fix alt text fallback

### PR DESCRIPTION
**Pull request by noam-heller1**
If the user didn't add alt text (in the media manager), then the alt text will be empty (instead of the image title/description/file name).

### CODE REVIEW

File: packages/gallery/src/components/item/imageItem.js [hunk 1]
```diff
@@ -227,9 +227,7 @@ class ImageItem extends React.Component {
           data-hook="gallery-item-image-img"
           data-idx={idx}
           src={src}
-          alt={
-            typeof calculatedAlt === 'string' ? calculatedAlt : 'untitled image'
-          }
+          alt={typeof alt === 'string' ? alt : 'untitled image'}
```
Comment (liatv) [inline packages/gallery/src/components/item/imageItem.js]: 'untitled image' shouldn't be ''?

File: packages/gallery/src/components/item/itemView.js [hunk 2]
```diff
@@ -779,12 +779,12 @@ class ItemView extends React.Component {
...
-    const { type, calculatedAlt, htmlContent, options } = this.props;
+    const { type, alt, htmlContent, options } = this.props;
     const mapTypeToLabel = {
       dummy: '',
       text: htmlContent,
-      video: calculatedAlt || 'Untitled video',
-      image: calculatedAlt || 'Untitled image',
+      video: alt || 'Untitled video',
+      image: alt || 'Untitled image',
     };
```
Comment (liatv) [inline packages/gallery/src/components/item/itemView.js:788]: 'Untitled image' / 'Untitled video' - shouldn't it be ''?

File: packages/layouts/src/classes/galleryItem.js [hunk 3]
```diff
@@ -691,22 +690,14 @@ class GalleryItem {
...
-  }
-
   get htmlContent() {
     return this.html?.replace(/<[^<>]*>/g, '').trim();
   }
 
   get alt() {
-    return this.metadata.alt || '';
+    return (
+      (utils.isMeaningfulString(this.metadata.alt) && this.metadata.alt) || ''
```
Comment (liatv) [inline packages/layouts/src/classes/galleryItem.js:699]: I think it should be 
(this.metadata.alt && utils.isMeaningfulString(this.metadata.alt)) || ''

**Review by liatv — APPROVED** [review]

File: packages/layouts/src/classes/galleryItem.js (hunk 3 above)
Comment (noam-heller1) [reply packages/layouts/src/classes/galleryItem.js:699]: It will return 'true' instead of the alt

**Review by IzaacAyelin — APPROVED** [review]

**Pull request merged**